cmd/create.go

```diff
@@ -259,6 +259,31 @@ to quickly create a Cobra application.`,
 		}
 		sendCompleteInstallTelemetry(dryRun)
 		time.Sleep(time.Millisecond * 100)
+
+		if dryRun {
+			log.Println("no handoff data on dry-run mode")
+			return
+		}
+
+		// prepare data for the handoff report
+		clusterData := reports.CreateHandOff{
+			ClusterName:       viper.GetString("cluster-name"),
+			AwsAccountId:      viper.GetString("aws.accountid"),
+			AwsHostedZoneName: viper.GetString("aws.hostedzonename"),
+			AwsRegion:         viper.GetString("aws.region"),
+			ArgoCDUrl:         viper.GetString("argocd.local.service"),
+			ArgoCDUsername:    viper.GetString("argocd.admin.username"),
+			ArgoCDPassword:    viper.GetString("argocd.admin.password"),
+			VaultUrl:          viper.GetString("vault.local.service"),
+			VaultToken:        viper.GetString("vault.token"),
+		}
```

Comment:
Do we want show or produce a json or yaml file so people can hook their automations on our post-installs?

Reply:
kind of.. that was my second point in the PR notes above. we can do it, but it would be better to provide some encryption layer to avoid plain text with sensitive data.